### PR TITLE
style(ui): add 💻 icon to peerId

### LINKED ISSUE #1259: Change @ icon to 💻  icon
Since the ID under the user's Display Name on the profile now represents the Device ID instead of the Radicle ID, I thought it would be more appropriate as a computer icon.

Here's a [link to the new icon](https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA/%F0%9F%8C%B1%F0%9F%86%99-Radicle-Upstream?node-id=6517%3A5).

It should be changed everywhere the @ icon was previously displayed. So it should be replaced in onboarding too, and anywhere else where we switched the Radicle ID to the Device ID.

image

### PR DESCRIPTION
**Pull request by juliendonck**
- changed in header
- changed in settings
- changed in onboarding

closes #1259

### CODE REVIEW

**Review by xla — CHANGES_REQUESTED** [review]
Is it intentional that the icon is not part of the copyable? Also can we move the whole thing into its own component, woul davoid repetiton.

**Comment by rudolfs** [comment]
> Is it intentional that the icon is not part of the copyable? Also can we move the whole thing into its own component, woul davoid repetiton.

It would make sense to have a new `PeerId` component for this. `StyledCopyable` could be re-used for other stuff as well that is not a `PeerId`...

**Comment by juliendonck** [comment]
> > Is it intentional that the icon is not part of the copyable? Also can we move the whole thing into its own component, woul davoid repetiton.
> 
> It would make sense to have a new `PeerId` component for this. `StyledCopyable` could be re-used for other stuff as well that is not a `PeerId`...

Ok

File: ui/DesignSystem/Primitive/Icon/Computer.svelte [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+  export let dataCy = "";
+  export let style = "";
+</script>
+
+<IconWrapper on:click {dataCy} {style}>
+  <path
+    fill-rule="evenodd"
+    clip-rule="evenodd"
+    d="M7 5C6.44771 5 6 5.44772 6 6V12H18V6C18 5.44772 17.5523 5 17 5H7ZM18.3406 14H5.65939L4.11393 17.6061C3.83113 18.2659 4.31516 19 5.03308 19H18.9669C19.6848 19 20.1689 18.2659 19.8861 17.6061L18.3406 14ZM4 12.7947V6C4 4.34315 5.34314 3 7 3H17C18.6569 3 20 4.34315 20 6V12.7947L21.7244 16.8182C22.5728 18.7978 21.1207 21 18.9669 21H5.03308C2.87933 21 1.42724 18.7978 2.27564 16.8182L4 12.7947Z" />
```
Comment (rudolfs) [inline ui/DesignSystem/Primitive/Icon/Computer.svelte:12]: @MeBrei is working on a permanent fix for our prettier config, meanwhile:

```suggestion
    d="M7 5C6.44771 5 6 5.44772 6 6V12H18V6C18 5.44772 17.5523 5 17
    5H7ZM18.3406 14H5.65939L4.11393 17.6061C3.83113 18.2659 4.31516 19 5.03308
    19H18.9669C19.6848 19 20.1689 18.2659 19.8861 17.6061L18.3406 14ZM4
    12.7947V6C4 4.34315 5.34314 3 7 3H17C18.6569 3 20 4.34315 20
    6V12.7947L21.7244 16.8182C22.5728 18.7978 21.1207 21 18.9669
    21H5.03308C2.87933 21 1.42724 18.7978 2.27564 16.8182L4 12.7947Z" />
```

**Review by rudolfs — APPROVED** [review]
Nice!

**Review by geigerzaehler — APPROVED** [review]

**Review by xla — DISMISSED** [review]
🤗 🌌 🕛 🌹

File: ui/DesignSystem/Primitive/Icon/Computer.svelte (hunk 1 above)
Comment (rudolfs) [reply ui/DesignSystem/Primitive/Icon/Computer.svelte:12]: Apparently it wasn't a straightforward fix, so we have an issue for it: https://github.com/radicle-dev/radicle-upstream/issues/1263.

**Review by geigerzaehler — APPROVED** [review]

**Review by xla — APPROVED** [review]
🏬 🍵 🏕 🔞

**Pull request merged**